### PR TITLE
[IMP] sale,*: update urls into nice urls

### PR DESCRIPTION
**Pull request by sauc-odoo**
*= sale_management, sale_project, sale_timesheet,
survey

This commit modifies existing URLs to the new format 
so that the URL's become more human readable.

Task-3820230

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/survey/views/survey_templates_management.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
                     <h1 class="mt32">403: Forbidden</h1>
                     <p>The page you were looking for could not be authorized.</p>
                     <p>Maybe you were looking for
-                        <a t-attf-href="/web#view_type=form&amp;model=survey.survey&amp;id=#{survey.id}&amp;action=survey.action_survey_form">this page</a> ?
+                        <a t-attf-href="/odoo/survey.action_survey_form/{survey.id}">this page</a> ?
```
Comment (jpp-odoo) [inline addons/survey/views/survey_templates_management.xml:16]:
```suggestion
                        <a t-attf-href="/odoo/action-survey.action_survey_form/{survey.id}">this page</a> ?
```

**Review by jpp-odoo — CHANGES_REQUESTED** [review]

File: addons/survey/views/survey_templates_management.xml (hunk 1 above)
Comment (sauc-odoo) [reply addons/survey/views/survey_templates_management.xml]: Hello @jpp-odoo 
Changes have been made, please check it.
Thanks :slightly_smiling_face:

File: addons/sale/controllers/portal.py [hunk 2]
```diff
@@ -154,10 +154,7 @@ def portal_order_page(
                     subtype_xmlid="sale.mt_order_viewed",
                 )
 
-        backend_url = f'/web#model={order_sudo._name}'\
-                      f'&id={order_sudo.id}'\
-                      f'&action={order_sudo._get_portal_return_action().id}'\
-                      f'&view_type=form'
+        backend_url = f'/odoo/action-{order_sudo._get_portal_return_action().id}/{order_sudo.id}'
```
Comment (Feyensv) [inline addons/sale/controllers/portal.py:157]: I don't find any mention of `sale_subscription` in the task, and the same change should be done there too.
There is a handful of `backend_url` here and there in the code, and I'm not sure all of them are considered in the scope mentioned in your linked task.

Comment (Feyensv) [reply addons/sale/controllers/portal.py:157]: Also, the link doesn't seem to work well if the website editor is enabled:
image

Comment (jpp-odoo) [reply addons/sale/controllers/portal.py:157]: The error is not related to the URL modification, the error already exists in master.

There is no mention of any module in the tasks, the PR are created module by module, a PR will be created for also for sale_subscription

Comment (Feyensv) [reply addons/sale/controllers/portal.py:157]: Didn't take the time to test in master but if it's unrelated, fine :ok_hand:.
For the modules, I thought that you made the table in advance to handle all the scopes, but if you do it step by step forget what I said.

**Comment by jpp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**